Makefile

```diff
@@ -46,24 +46,38 @@ run-simple-system: sw-simple-hello | $(Vibex_simple_system)
 		--raminit=$(simple-system-program)
 
 
-# Arty A7 100T
-# Use the following targets:
-# - "build-arty"
-# - "program-arty"
+# Arty A7 FPGA example
+# Use the following targets (depending on your hardware):
+# - "build-arty-35"
+# - "program-arty-35"
+# - "build-arty-100"
+# - "program-arty-100"
 arty-sw-program = examples/sw/led/led.vmem
 sw-led: $(arty-sw-program)
 
 .PHONY: $(arty-sw-program)
 $(arty-sw-program):
 	cd examples/sw/led && $(MAKE)
 
-build-arty: sw-led
+.PHONY: build-arty-35
+build-arty-35: sw-led
+	fusesoc --cores-root=. run --target=synth --setup --build \
+		lowrisc:ibex:top_artya7 --part xc7a35ticsg324-1L
+
+.PHONY: build-arty-100
+build-arty-100: sw-led
 	fusesoc --cores-root=. run --target=synth --setup --build \
-		lowrisc:ibex:top_artya7_100
+		lowrisc:ibex:top_artya7  --part xc7a100tcsg324-1
+
+.PHONY: program-arty-35
+program-arty-35:
+	fusesoc --cores-root=. run --target=synth --run \
+		lowrisc:ibex:top_artya7 --part xc7a35ticsg324-1L
```

Comment:
For the program (run --run) step, I think it is not required to specify `--part`. If you can confirm this, I think it is best to omit it here.

Reply:
Hm yes. Not sure why I have added that actually anymore :( I was trying lots of things till I settled on the current approach. I will remove the individual program recipes again because there is no difference between both versions. However, this is also a potential problem: if anyone got both boards and tries to build and program them both... I guess we (should) ignore that but I wanted to mention it.

Reply:
Yeah, building both won't be possible, but I think we can ignore that. A solution for this might depend on FuseSoC anyway.
Building for one and then programming the other board should result in an Vivado error. So at least it will fail with an indication.

Reply:
Yes, `fusesoc` uses the information of the synthesis and supplies that to Vivado for the download (even if you specify the `--part` to presumably force it fusesoc ignores it). Vivado then checks for the ID and prints an error that they don't match (IIRC).